TestAssets/TestProjects/TestAppWithMigrateAbleScripts/Program.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
```

Comment:
This is a weird name for this project. MigrateAble. Did you mean Migrateable? Right now, I read Migrate Able. Intentional?